retry/src/main/java/io/micronaut/retry/annotation/Retryable.java

```diff
@@ -91,4 +91,10 @@
      * @return The capture exception types (defaults to RuntimeException)
      */
     Class<? extends Throwable> capturedException() default RuntimeException.class;
+
+    /**
+     * @return The jitter factor used to apply random deviation to retry delays
+     */
+    @Digits(integer = 1, fraction = 2)
+    String jitter() default "0.25";
```

Comment:
```suggestion
    String jitter() default "0.0";
```
should be set default jitter to `0`? (jitter `0` means don't apply jitter by default)

Reply:
That's a valid point. Thank you for the suggestion!